### ISSUE TITLE
Tracking the user on startup app

### ISSUE THREAD

**Issue by FabianHofman** [Wykks/ngx-mapbox-gl#78]
Is there a way to set the location tracker on when the application loads?
I have no idea how it works.

I am using the NgxMapboxGLModule and the @Types for this.

Thanks in advance!

**Comment by Wykks** [comment]
Hi!

This ? https://wykks.github.io/ngx-mapbox-gl/demo/edit/locate-user

**Comment by FabianHofman** [comment]
Yes, but i want the button to be pressed when the application starts.
Without the user to interact with it!

**Comment by Wykks** [comment]
Like this : https://stackblitz.com/edit/angular-y89zlb?file=demo.ts

(TODO for me: expose ControlComponent to avoid the any type)

**Comment by FabianHofman** [comment]
Yes!!
Thank you very much!

**Comment by LarBearrr** [comment]
Can you provide the source for the `demo.js` is the link seems to be broken. I am trying to achieve the same result, by tracking the user location when my app/page is loaded without having to press the locate user button. I can successfully use the geolocation api to get the popup, but I haven't figured out how to update the map position with these coordinates.
